mysql-test/suite/galera/t/galera_drop_if_exists.test

```diff
@@ -0,0 +1,30 @@
+# Check that the following requirement is fulfiled
+# The DROP DATABASE IF EXISTS, DROP TABLE IF EXISTS, DROP VIEW IF EXISTS and DROP EVENT IF EXISTS statements are always replicated,
+# even if the database, table, or view to be dropped does not exist on the source.
+# This is to ensure that the object to be dropped no longer exists on either the source or the replica,
+# once the replica has caught up with the source.
+#
+# Additionally check that such statements generate cluster gtid instead of local gtid.
+
+--source include/galera_cluster.inc
+--source include/gtid_utils.inc
+
+--connection node_1
+--source include/gtid_step_reset.inc
+DROP DATABASE IF EXISTS db1;
+DROP TABLE IF EXISTS t1;
+DROP VIEW IF EXISTS v1;
+DROP EVENT IF EXISTS ev1;
+
+# check that GTID is advanced by 4
+--let $gtid_step_count = 4
+--let $gtid_step_uuid = `SELECT LEFT(@@global.gtid_executed, 36);`
```

Comment:
With this we don't ensure that the events are binlogged with cluster uuid. Can we instead try this way?
```suggestion
--let $gtid_step_uuid = `SELECT Variable_value FROM performance_schema.global_status WHERE Variable_name='wsrep_cluster_state_uuid'`
```